### DISCUSSION TITLE
Intellisense is slow for numpy, pandas and similar

### DISCUSSION THREAD

**Discussion by dlefcoe** [microsoft/pylance-release#1142] [Q&A]
I have imported pandas the standard way:
import pandas as pd
When I start coding, intellisense in vscode works fine until i import the pandas module and then it starts to go very slow.

please see my original post before i was directed here:
[https://github.com/microsoft/vscode-python/discussions/15892#discussioncomment-590790](url)

**Comment by jakebailey** [comment]
Can you please file a real issue with our bug template filled out? We'd like to be able to see details about your situation and a discussion isn't the right mechanism for it. Thanks.

**Reply by dlefcoe** [reply]
where does one file a "real issue" please ?

**Reply by jakebailey** [reply]
Here is a direct link to the bug template: https://github.com/microsoft/pylance-release/issues/new?assignees=&labels=&template=bug-report.md&title=
